### PR TITLE
Adding United Cloud to adopters list

### PR DESCRIPTION
**Pull request by carnerito**
Signed-off-by: Boris Petrovic <carnerito.b@gmail.com>

### CODE REVIEW

File: USERS.md [hunk 1]
```diff
@@ -561,6 +561,12 @@ Users (Alphabetically)
       U: ClusterMesh, CNI-Chaining (with amazon-vpc-cni-k8s)
       Q: @jirving
 
+    * N: United Cloud
+      D: United Cloud is using Cilium for all non-production and production clusters (on-premises)
+      U: CNI, Hubble, CiliumNetworkPolicy, CiliumClusterwideNetworkPolicy, ClusterMesh, Encryption
+      L: https://united.cloud
+      Q: @boris
```
Comment (pchaigno) [inline USERS.md:568]: I think this is usually a GitHub or Twitter handle, hence the `@`.

Comment (carnerito) [reply USERS.md:568]: @pchaigno Sorry, I didn't get it. Do I have to change something?

Comment (pchaigno) [reply USERS.md:568]: Yes, I think you want to change `@boris` to `@carnerito` if the goal was to set yourself as the point of contact.

Comment (carnerito) [reply USERS.md:568]: `@boris` is my handle on Cilium Slack. Do I need to emphasize that?

Comment (pchaigno) [reply USERS.md:568]: Oh, right :facepalm: The example even suggests using such a Slack handle. My bad. Sorry about that.

**Review by pchaigno — APPROVED** [review]
LGTM.

**Pull request merged**